RPIPs/rpip-balance-sub-guard.md

```diff
@@ -0,0 +1,52 @@
+---
+rpip: #<to be assigned>
+title: Balance Submission Guardrail
+description: <Description is one full (short) sentence>
+author: knoshua (@knoshua)
+discussions-to: https://dao.rocketpool.net/t/rpip-balance-submission-guardrail/3010
+status: Draft
+type: Protocol
+category (*only required for Protocol ): Core
+created: 2024-05-27
+
+---
+
+## Abstract
+This proposal limits how much the oDAO can change the rETH rate in a single update. The limit is a protocol parameter controlled by the pDAO. The pDAO can't set it lower than a minimum value. The parameter can't be changed by the Security Council.
+This proposal also limits how frequently updates can happen, using an existing protocol parameter that the pDAO controls. 
+
+## Motivation
+Currently, the oDAO can change rETH rate arbitrarily within one block. This exposes rETH holders: In the worst case, a compromised oDAO can make rETH worthless for any holder and extract any ETH available. During normal operation, the rETH rate doesn't need to change a lot. This proposal aims to reduce oDAO trust by introducing a sensible limit that doesn't interfere with legitimate use of balance updates and removes unnecessary risk from illegitimate use.
+
+## Specification
+
+### Maximum rETH Delta Protocol Parameter
+- There SHALL be a Maximum rETH Delta protocol parameter that the pDAO can vote to change.
+- Maximum rETH Delta SHALL initially be set to 2 percent.
+- The pDAO SHALL NOT be able to set Maximum rETH Delta lower than 1 percent.
+- The Security Council SHALL NOT be able to change Maximum rETH Delta.
+
+### Network Balance Updates
+- The oDAO SHALL NOT be able to be submit a Network Balance update if the time elapsed since the last Network Balance update is less than the Network Submit Balances Frequency.
```

Comment:
I'm a little late to the party... but shouldn't this be about target block rather than submission time?

An attempt:
```
- The oDAO SHALL NOT be able to be submit a Network Balance update if the time elapsed between the target block and the last successful Network Balance update's target block is less than the Network Submit Balances Frequency.
```

Reply:
If we go to target block as suggested, this opens the door to:
- no updates happen for a a week
- within one block, 7 updates happen with target blocks Monday, Tuesday, Wednesday, ...

So I do think I prefer directly limiting how often submissions can happen.